src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/CaseWorkerRefController.java

```diff
@@ -90,16 +86,11 @@
             consumes = "multipart/form-data")
     @Secured("cwd-admin")
     public ResponseEntity<Object> caseWorkerFileUpload(@RequestParam("file")  MultipartFile file) {
-        Workbook workbook = excelValidatorService.validateExcelFile(file);
-        excelAdaptorService.parseExcel(workbook, uk.gov.hmcts.reform.cwrdapi.client.domain.CaseWorkerProfile.class);
-
-        UploadCaseWorkerFileResponse uploadCaseWorkerFileResponse = UploadCaseWorkerFileResponse.builder()
-                .message(CaseWorkerConstants.REQUEST_COMPLETED_SUCCESSFULLY)
-                .build();
-
-        return ResponseEntity
-                .status(201)
-                .body(uploadCaseWorkerFileResponse);
+        long time1 = System.currentTimeMillis();
+        ResponseEntity<Object> responseEntity = caseWorkerServiceFacade.processFile(file);
+        log.info("----Time taken to upload the given file "
+                + (System.currentTimeMillis() - time1));
```

Comment:
{} use placeholders rather + and also please use logging component name to track this in app insights

Reply:
sure